stable/grafana/templates/deployment.yaml

```diff
@@ -13,6 +13,8 @@ spec:
   template:
     metadata:
       annotations:
+        checksum/server-config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
Yes, I though about it, except this chart has two configs, and the other charts only have one config.